app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,6 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
-  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze
+
+  def initialize(cfg = {})
+    super(cfg)
+
+    # Accept events with an event category in the keyset and whose event message matches the (optional) pattern value
+    # - key = event category
+    # - value (optional) event message regex
+    @whitelist = {
+      'Hawkular Deployment' => nil
```

Comment:
tbh, I'd always add a Regexp as a value and not make it optional.
The regex are precompiled, so perfomance impact is negligible, but you improve on the readability of `def whitelist?` below